docs/TBG_macros.cfg

```diff
@@ -155,7 +155,7 @@ TBG_JField_slice="--J_slice.period 100 --J_slice.fileName sliceJ --J_slice.plane
 # Sum up total energy every .period steps for
 # - species   (--<species>_energy)
 # - fields    (--fields_energy)
-TBG_sumEnergy="--fields_energy.period 10 --<species>_energy.period 10"
```

Comment:
using the specific filter `all` instead of `<filter name>` would make sense here since the tbg var is called `sumEnergy`

in other plugins, I would use `<filterName>` as the spelling for the place holder

Reply:
done, This comment is pinned to a deleted source code line